dot/telemetry/afg_authority_set.go

```diff
@@ -3,25 +3,25 @@
 
 package telemetry
 
-// afgAuthoritySetTM is a telemetry message of type `afg.authority_set` which is
+// AfgAuthoritySetTM is a telemetry message of type `afg.authority_set` which is
```

Comment:
Ensure at compilation it implements the interface.

```suggestion
var _ Message = (*AfgAuthoritySetTM)(nil)

// AfgAuthoritySetTM is a telemetry message of type `afg.authority_set` which is
```

Reply:
Added!